UOP1_Project/Assets/Scripts/Characters/Character.cs

```diff
@@ -23,6 +24,7 @@ public class Character : MonoBehaviour
     private float verticalMovement = 0f; //Represents how much a player will move vertically in a frame. Affected by gravity * gravityContributionMultiplier
     private Vector3 inputVector; //Initial input horizontal movement (y == 0f)
     private Vector3 movementVector; //Final movement vector
+    private float rotationThreshold = .02f; // Used to prevent NaN result causing rotation in a non direction
```

Comment:
if it's not being changed on inspector, you can probably set it as a 
`private const float RotationThreshold`

Reply:
Good point!